### ISSUE TITLE
Missing depencencies for the official Python wheel (or, lacking documentation?)

### ISSUE THREAD

**Issue by xor-xor** [microsoft/onnxruntime#9966]
**Describe the bug**
Having a newly-created virtualenv with the latest version of `onnxruntime` installed in it via `pip install onnxruntime`, some imports fail because of missing dependencies:

```
>>> from onnxruntime.quantization import quantize_dynamic
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/xor-xor/envs/my-test-env/lib/python3.7/site-packages/onnxruntime/quantization/__init__.py", line 1, in <module>
    from .quantize import quantize, quantize_static, quantize_dynamic, quantize_qat
  File "/Users/xor-xor/envs/my-test-env/lib/python3.7/site-packages/onnxruntime/quantization/quantize.py", line 7, in <module>
    import onnx
ModuleNotFoundError: No module named 'onnx'
```

```
>>> from onnxruntime.transformers import optimizer
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/xor-xor/envs/my-test-env/lib/python3.7/site-packages/onnxruntime/transformers/optimizer.py", line 21, in <module>
    import coloredlogs
ModuleNotFoundError: No module named 'coloredlogs'
```

The latter seems to be a known one (see #7527), although the context is slightly different, as I'm not building from source here. 

Installing those missing packages manually solves the problem, but if that's the recommended way of proceeding, then I think it should be mentioned [here](https://onnxruntime.ai/docs/install/#python-installs) or [here](https://onnxruntime.ai/docs/install/#requirements).

**Urgency**
None.

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): macOS 10.15.7
- ONNX Runtime installed from (source or binary): binary
- ONNX Runtime version: 1.10.0
- Python version: 3.7.7
- Visual Studio version (if applicable): n/a
- GCC/Compiler version (if compiling from source): n/a
- CUDA/cuDNN version: n/a
- GPU model and memory: n/a

**To Reproduce**
1. `python -m venv my-test-env`
2. `source my-test-env/bin/activate`
3. `pip install pip --upgrade && pip install onnxruntime`
4. `python -c "from onnxruntime.quantization import quantize_dynamic"`
5. `python -c "from onnxruntime.transformers import optimizer"`

**Expected behavior**
Imports from steps (4) and (5) would succeed.

**Screenshots**
n/a

**Additional context**
n/a

**Comment by natke** [comment]
Fixed by #9969

**Comment by xor-xor** [comment]
@natke I'm failing to see how my issue is fixed by #9969. In particular, why do you write [here](https://onnxruntime.ai/docs/get-started/with-python.html#install-onnx-for-model-export) that "ONNX is built into PyTorch" suggesting that `pip install torch` would be enough to get `onnx` package (as a dependency of PT, as one may assume)? Observe:
```
$ python --version
Python 3.9.15

$ pip freeze | grep onnx
onnxruntime==1.13.1

$ pip freeze | grep torch
torch==1.13.1

$ python -c "from onnxruntime.transformers import optimizer"
[...]
ModuleNotFoundError: No module named 'onnx'

$ python -c "from onnxruntime.quantization import quantize_dynamic"
[...]
ModuleNotFoundError: No module named 'onnx'

$ python -c "import torch; print(torch.onnx)"
<module 'torch.onnx' from '/Users/xor-xor/envs/ort-test/lib/python3.9/site-packages/torch/onnx/__init__.py'>
# ^ that's not onnx the _package_ - it's a module for exporting PT models into ONNX format
```
